packages/forms/backend/sirius-components-collaborative-widget-reference/src/main/java/org/eclipse/sirius/components/collaborative/widget/reference/browser/ModelBrowserMetadataProvider.java

```diff
@@ -26,19 +28,17 @@
 public class ModelBrowserMetadataProvider implements IRepresentationMetadataProvider {
 
     @Override
-    public boolean canHandle(String representationId) {
-        return representationId != null && (representationId.startsWith(ModelBrowsersDescriptionProvider.MODEL_BROWSER_CONTAINER_PREFIX) || representationId.startsWith(ModelBrowsersDescriptionProvider.MODEL_BROWSER_REFERENCE_PREFIX));
-    }
-
-    @Override
-    public RepresentationMetadata handle(String representationId) {
+    public Optional<RepresentationMetadata> handle(String representationId) {
+        if (representationId == null) {
+            return Optional.empty();
+        }
```

Comment:
Why would that be `null`? I'll look into it but it should be the responsibility of the caller to give us proper parameters

Reply:
Done